wadsrc/static/zscript/shared/sharedmisc.txt

```diff
@@ -159,7 +159,7 @@ class CustomSprite : Actor
 		String name;
 		Super.BeginPlay ();
 
-		format(name, "BTIL%04d", args[0] & 0xffff);
+		//format(name, "BTIL%04d", args[0] & 0xffff); // note: this does NOTHING and did NOTHING. what?
 		picnum = TexMan.CheckForTexture (name, TexMan.TYPE_Build);
```

Comment:
I have no fucking idea what that line does.
The syntax for format is "format(<format string>, <format arguments>)".
Graf probably meant name = format("BTIL%04d", args[0]&0xffff), but I'm not sure about that and didn't even want to touch it — Graf can do it himself.